Sources/BottomSheet/IsPresentedBottomSheet.swift

```diff
@@ -0,0 +1,79 @@
+//
+//  IsPresentedBottomSheet.swift
+//  BottomSheet
+//
+//  Created by Sam Doggett on 10/28/24.
+//
+
+import SwiftUI
+
+struct IsPresentedBottomSheet<Content: View, SheetContent: View>: BottomSheet {
```

Comment:
"IsPresentedBottomSheet" is very odd struct name for me. Why not just BottomSheet and BottomSheetProtocol?